polyglot/piranha/src/utilities/mod.rs

```diff
@@ -92,6 +99,9 @@ pub(crate) fn find_file(input_dir: &PathBuf, name: &str) -> PathBuf {
     .path()
 }
 
+
+
+
```

Comment:
What are these newlines for?

Reply:
I Am going to see how to run cargo fmt each time before the code gets to review.

Reply:
Have you considered setting it up as a git pre-commit hook and adding a CI job to check for it on PRs?